wiki/docs/message/placeholder.md

```diff
@@ -0,0 +1,51 @@
+# Placeholders
+JDA-Commands supports resolving named placeholders/variables in all places where [localization](localization.md#implicit-localization) is done implicitly.
```

Comment:
while technically the truth, this is confusing, because this conflicts with the wording of overview#Implicit resolution

Reply:
I think this should either link to the overview or list the places again

Reply:
Ah yes, that's an oversight. Should link to the overview.